pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[project]
+# Project metadata. Available keys are documented at:
+# https://packaging.python.org/en/latest/specifications/declaring-project-metadata
+name = "pathwaysutils"
+readme = "README.md"
+requires-python = ">=3.10"
+license = {file = "LICENSE"}
+authors = [{name = "Pathways-on-Cloud Utilities Developers", email="pathwaysutils-dev@google.com"}]
+classifiers = [  # List of https://pypi.org/classifiers/
+    "License :: OSI Approved :: Apache Software License",
+]
+keywords = []
+
+dependencies = [
+    "absl-py",
+    "fastapi",
+    "google-cloud-logging",
+    "jax>=0.4.26",
+    "orbax-checkpoint",
+    "uvicorn",
+]
+
+# `version` is automatically set by flit to use `my_project.__version__`
```

Comment:
I assume flit makes our life easier? Just wanted to understand why we are taking this dep.

Reply:
The switch from setup.py to pyproject.toml makes PyPI metadata management a little easier, e.g., readme/version/description being easily deducted from code.

Out of the different build chains, flit is maintained by pypa and meets our needs. In their own words, https://flit.pypa.io/en/stable/rationale.html